### DISCUSSION TITLE
Custom validation message on TextInputColumn on RelationManager

### DISCUSSION THREAD

**Discussion by jwhoami** [filamentphp/filament#6240] [Help]
Hi,

I have a relation manager with a textcolumn like this:

`Tables\Columns\TextInputColumn::make('qty')
          ->validationAttribute('custom_qty')
          ->rules(fn ($record) => ['min:1', 'max:' . $record->max_qty]),`

I don't want the default validation message but want to show a custom message 
like "You should order a min qty of 1"

How can I achieve this?

Thanks

**Comment by jwhoami** [comment]
Please help

I tried adding custom validation custom message to the validation.php its still showing the default message.

`  'custom' => [
    'custom_qty' => [
      'min' => 'HEY',
      'max' => 'HEY',
    ],
  ],`

Thanks

**Reply by fvanzeijl** [reply]
Filament wraps all fields in a data-object.
So you have to prepend the error-key with `data.`.

```php 
 'custom' => [ 'data.custom_qty' => [ 'min' => 'HEY', 'max' => 'HEY', ], ],
 ```

**Reply by luisprmat** [reply]
This doesn't work!
